### PR TITLE
Remove oracle support

### PR DESCRIPTION
**Pull request by janosSarusiKis**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     |yes
| Deprecations?   |yes
| Related tickets | 
| License         | Apache 2.0

### What's in this PR?
Colmplete remove of Oracle support. 

- Oracle and related packages removed
- Dependent code removed or updated
- Migrations added

### Checklist

- [x] Implementation tested (with at least one cloud provider)
- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] OpenAPI and Postman files updated (if needed)
- [x] User guide and development docs updated (if needed)

### CODE REVIEW

**Review by sagikazarmark — APPROVED** [review]
Please wait for one more reviewer to approve.

**Review by orymate — APPROVED** [review]

**Review by pregnor — APPROVED** [review]
Make sure to communicate the breaking change in the release notes.

**Pull request merged**
